services/dao-service.ts

```diff
@@ -96,6 +99,48 @@ export default class DAO {
     return token;
   }
 
+
+  transactionHandler(event: PromiEvent<TransactionReceipt | Contract>,
+    resolve: ResolveReject,
```

Comment:
these arguments are not respecting code-style-spacing , they should be aligned with "event"